transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -58,6 +59,23 @@ public static synchronized HyperwalletTransferMethodUi getInstance(
         return sInstance;
     }
 
+    /**
+     * Returns an instance that will also initialize the Insights library.
+     *
+     * @param context                     A Context of the application consuming this Intent.
+     * @param authenticationTokenProvider An implementation of the {@link HyperwalletAuthenticationTokenProvider}
+     * @return singleton instance of HyperwalletTransferMethodUi
+     */
+    public static synchronized HyperwalletTransferMethodUi getInstance(@NonNull final Context context,
```

Comment:
are we going to keep both getInstance methods?

Reply:
for now yes, but @dnip-hw please add @deprecated on the other ones thanks, we will be removing this from docs and keeping it until we moved all UI instrumented to new `getInstance`